packages/rs-drive-abci/src/test/helpers/setup.rs

```diff
@@ -131,6 +133,12 @@ impl TempPlatform<MockCoreRPCLike> {
             )
             .expect("should create root tree successfully");
 
+        let block_info = BlockInfo::default_with_current_time();
+
+        let mut platform_state = self.platform.state.write().unwrap();
+        platform_state.set_genesis_block_info(Some(block_info));
+        drop(platform_state);
```

Comment:
I think this is an actual change from before, why is it needed?

Reply:
Some tests aren't passing that does createAt/updatedAt validation.

Reply:
Reverted anyways